docs/getting_started/install_ez_platform.md

```diff
@@ -7,31 +7,33 @@
     To install eZ Platform for development on macOS or Windows,
     see [Install on macOS or Windows](../community_resources/installing-on-mac-os-and-windows.md).
 
-## Prepare work environment
+## Prepare the work environment
 
 To install eZ Platform you need a stack with your operating system, MySQL and PHP.
 
 You can install it by following your favorite tutorial, for example: [Install LAMP stack on Ubuntu](https://www.digitalocean.com/community/tutorials/how-to-install-linux-apache-mysql-php-lamp-stack-ubuntu-18-04).
 
-Additionally, you need [Node.js](https://nodejs.org/en/) and [Yarn](https://yarnpkg.com/lang/en/docs/install/#debian-stable) for asset management.
+Additional requirements:
 
-[For production](#prepare-installation-for-production) you also need Apache or nginx as the HTTP server (Apache is used as an example below).
-[For development](#use-phps-built-in-server), you can use the built-in PHP server instead.
+- [Node.js](https://nodejs.org/en/) and [Yarn](https://yarnpkg.com/lang/en/docs/install/#debian-stable) for asset management.
+- `git` for version control.
+- to use search in the shop front end, you need to have [Solr installed](#install-solr).
 
-You also need `git` for version control.
+[For production](#prepare-installation-for-production) you also need Apache or nginx as the HTTP server (Apache is used as an example below).
 
 Before getting started, make sure you review the [requirements](requirements.md) page to see the systems we support and use for testing.
```

Comment:
The reqs in this page are actually a subset of those on the Requirements page, so this makes more sense:
```suggestion
Before getting started, make sure you review other [requirements](requirements.md) to see the systems we support and use for testing.
```